src/test/suite/fixtures/recorded/languages/rust/chuckValueAir.yml

```diff
@@ -0,0 +1,36 @@
+languageId: rust
+command:
+  spokenForm: chuck value air
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: a}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: value}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    match user {
+        User { first_name: "John", last_name: last_name } => {}
```

Comment:
Ditto - I know it should be deleting `: last_name` but the test doesn't explicitly test that!